dev/archery/archery/integration/datagen.py

```diff
@@ -1541,6 +1637,15 @@ def generate_run_end_encoded_case():
     return _generate_file("run_end_encoded", fields, batch_sizes)
 
 
+def generate_view_case():
```

Comment:
Perhaps `generate_binary_view_case` since we'll probably make a separate case for list view.